data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+          ["ironhead"],
+          ["swordsdance"]
+        ],
+        "item": ["blackglasses", "sitrusberry"],
+        "nature": "adamant",
+        "evs": [252, 252, 0, 0, 0, 4],
+        "teratype": ["dark", "flying"],
+        "ability": "defiant"
+      },
+      {
+        "species": "kingambit",
+        "weight": 5,
+        "moves": [
+          ["stealthrock"],
+          ["kowtowcleave"],
+          ["ironhead", "guillotine"],
+          ["suckerpunch", "thunderwave"]
+        ],
+        "item": ["sitrusberry", "focussash"],
+        "nature": "adamant",
+        "evs": [244, 252, 4, 0, 4, 4],
+        "teratype": ["flying", "fairy", "dark"],
+        "ability": "defiant"
+      },
+      {
+        "species": "kingambit",
+        "weight": 5,
+        "moves": [
+          ["stealthrock"],
+          ["kowtowcleave"],
+          ["ironhead", "guillotine"],
+          ["suckerpunch", "thunderwave"]
+        ],
+        "item": ["sitrusberry", "focussash"],
+        "nature": "adamant",
+        "evs": [244, 252, 4, 0, 4, 4],
+        "teratype": ["flying", "fairy", "dark"],
+        "ability": "supremeoverlord"
+      }
+    ]
+  },
+  "volcarona": {
+    "weight": 5,
+    "sets": [
+      {
+        "species": "volcarona",
+        "weight": 55,
+        "moves": [
+          ["quiverdance"],
+          ["fierydance"],
+          ["morningsun"],
+          ["gigadrain", "willowisp", "substitute"]
+        ],
+        "item": ["heavydutyboots", "sitrusberry"],
+        "nature": "timid",
+        "evs": [244, 0, 204, 12, 4, 44],
+        "teratype": ["fairy", "grass"],
+        "wantsTera": true,
+        "ability": "flamebody"
+      },
+      {
+        "species": "volcarona",
+        "weight": 25,
+        "moves": [
+          ["quiverdance"],
+          ["fierydance"],
+          ["morningsun"],
+          ["terablast"]
+        ],
+        "item": ["heavydutyboots", "sitrusberry"],
+        "nature": "timid",
+        "evs": [244, 0, 204, 12, 4, 44],
+        "teratype": ["water"],
+        "wantsTera": true,
+        "ability": "flamebody"
+      },
+      {
+        "species": "volcarona",
+        "weight": 20,
+        "moves": [
+          ["quiverdance"],
+          ["fierydance"],
+          ["gigadrain", "bugbuzz", "psychic"],
+          ["terablast"]
+        ],
+        "item": ["heavydutyboots", "lumberry", "sitrusberry"],
+        "nature": "timid",
+        "evs": [0, 0, 0, 252, 4, 252],
+        "teratype": ["water", "rock"],
+        "wantsTera": true,
+        "ability": "flamebody"
+      }
+    ]
+  },
+  "wochien": {
+    "weight": 5,
+    "sets": [
+      {
+        "species": "wochien",
+        "weight": 90,
+        "moves": [
+          ["leechseed"],
+          ["protect"],
+          ["foulplay", "knockoff", "darkpulse"],
+          ["substitute", "ruination", "gigadrain", "taunt"]
+        ],
+        "item": ["leftovers"],
+        "nature": "bold",
+        "evs": [252, 0, 252, 0, 0, 4],
```

Comment:
Please don't put the 4 in Speed on max/max sets in randoms. Move it elsewhere. Also applies to other Wo-Chiens.